resalloc_agent_spawner/worker.py

```diff
@@ -37,7 +37,7 @@ def handle_ticket(self, ticket_id):
         # We know there's self._redis initialized by parent class so we don't
         # create yet another connection.
         redis_dict = self._redis.hgetall(redis_key)
-        ticket_data = base64.b64encode(redis_dict["data"])
+        ticket_data = base64.b64encode(bytes(redis_dict["data"], 'utf-8'))
```

Comment:
I need to take a look ... it feels weird taking a string, encoding it tu utf, and then base64... we do something wrong